moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -30,4 +30,9 @@
     </description>
   </class>
 
+  <class name="default_planner_request_adapters/AddIterativeSplineParameterization" type="default_planner_request_adapters::AddIterativeSplineParameterization" base_class_type="planning_request_adapter::PlanningRequestAdapter">
```

Comment:
Good catch.  I've changed the cpp file to match instead to avoid colliding with #382 .